translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"wle 'print bpi(2000)'\n"
+"\n"
+"\t\t# Start the cron job to compute π to 2000 places and print it out every "
+"5 minutes.\n"
+"\t\tkubectl run pi --schedule=\"0/5 * * * ?\" --image=perl --"
+"restart=OnFailure -- perl -Mbignum=bpi -wle 'print bpi(2000)'"
+msgstr ""
+"\n"
+"\t\t# Inicia uma única instância de nginx.\n"
+"\t\tkubectl run nginx —image=nginx\n"
+"\n"
+"\t\t# Inicia uma única instância do hazelcast e expõe a porta 5701 do "
+"container.\n"
+"\t\tkubectl run hazelcast —image=hazelcast —port=5701\n"
+"\n"
+"\t\t# Inicia uma única instância do hazelcast e seta as variáveis de "
+"ambientente “DNS_DOMAIN=cluster” e “POD_NAMESPACE=default” no container.\n"
+"\t\tkubectl run hazelcast —image=hazelcast —env=“DNS_DOMAIN=cluster” —"
+"env=“POD_NAMESPACE=default”\n"
+"\n"
+"\t\t# Inicia uma instância replicada de nginx.\n"
+"\t\tkubectl run nginx —image=nginx —replicas=5\n"
+"\n"
+"\t\t# Dry run. Mostra os objetos da API correspondente sem criar elas.\n"
+"\t\tkubectl run nginx —image=nginx —dry-run\n"
+"\n"
+"\t\t# Inicia uma única instância de nginx, mas sobreescreve a spec do "
+"deployment com um Set parcial de valores passeados do JSON.\n"
+"\t\tkubectl run nginx —image=nginx —overrides=‘{ “apiVersion”: “v1”, “spec”: "
+"{ … } }’\n"
+"\n"
+"\t\t# Inicia um pod de busybox e mantém ele em primeiro plano, não reinicia "
+"se ele já existe.\n"
+"\t\tkubectl run -i -t busybox —image=busybox —restart=Never\n"
+"\n"
+"\t\t# Inicia um container nginx usando o comando padrão, mas utiliza "
+"argumentos customizados (arg1 .. argN) para o comando.\n"
+"\t\tkubectl run nginx —image=nginx — <arg1> <arg2> … <argN>\n"
+"\n"
+"\t\t# Inicia um container nginx usando um comando diferente e argumentos "
+"customizados.\n"
+"\t\tkubectl run nginx —image=nginx —command — <cmd> <arg1> … <argN>\n"
+"\n"
+"\t\t# Inicia um container perl para processar π to 2000 posições e mostra a "
+"saída.\n"
+"\t\tkubectl run pi —image=perl —restart=OnFailure — perl -Mbignum=bpi -wle "
+"‘print bpi(2000)’\n"
+"\n"
+"\t\t# Inicia um cron job para processar π to 2000 posições e mostra a saída "
+"a cada 5 minutos.\n"
+"\t\tkubectl run pi —schedule=“0/5 * * * ?” —image=perl —restart=OnFailure — "
+"perl -Mbignum=bpi -wle ‘print bpi(2000)’"
+
+#: pkg/kubectl/cmd/taint.go:67
+msgid ""
+"\n"
+"\t\t# Update node 'foo' with a taint with key 'dedicated' and value 'special-"
+"user' and effect 'NoSchedule'.\n"
+"\t\t# If a taint with that key and effect already exists, its value is "
+"replaced as specified.\n"
+"\t\tkubectl taint nodes foo dedicated=special-user:NoSchedule\n"
+"\n"
+"\t\t# Remove from node 'foo' the taint with key 'dedicated' and effect "
+"'NoSchedule' if one exists.\n"
+"\t\tkubectl taint nodes foo dedicated:NoSchedule-\n"
+"\n"
+"\t\t# Remove from node 'foo' all the taints with key 'dedicated'\n"
+"\t\tkubectl taint nodes foo dedicated-"
+msgstr ""
+"\n"
+"\t\t# Atualiza o nodo ‘foo’ com um taint com a chave ‘dedicated’ e o valor "
```

Comment:
we used the word "node" in the whole translation, can't see why not use here :)

Also, I think we can translate the taint word to "restrição", like "Atualiza o node 'foo' com uma restrição para a chave [...]